iroh-bytes/src/util.rs

```diff
@@ -129,6 +129,38 @@ impl HashAndFormat {
     }
 }
 
+impl Display for HashAndFormat {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        let mut slice = [0u8; 64];
+        hex::encode_to_slice(self.hash.as_bytes(), &mut slice).unwrap();
+        write!(f, "{}", std::str::from_utf8(&slice).unwrap())?;
+        if self.format.is_hash_seq() {
+            write!(f, "s")?;
```

Comment:
why use a suffix, aren't we usually using prefixes for things like this?

Reply:
prefix works as well. Just needed something.